src/utils/utils.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, test } from "@jest/globals";
+import validateEventData from "./validateEventData";
+
+describe("validation of eventData", () => {
+  const eventData = {
```

Comment:
I think the interface PageViewParametersInterface is compatible with this object. You can use it to add a type to eventData

Reply:
Code rewritten, resolved